pitch.md

```diff
@@ -23,38 +89,174 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+The users should write their schedule in a text file or provide an .ical file
+(generated from Google calendar or apple calendar, for users who do not want 
+ or need the increased expressiveness and concision provided by my DSL, but
+ would still like to schedule with others who **do** use the DSL).
+
+Here are some sample descriptions that I am considering:
+
+```
+time zone: PST
+
+On Monday, Wednesday, Friday:
+  busy 9am - 11am
+  busy 12pm - 12:30pm
+  busy 3pm - 4pm
+
+On Thursday:
+  free one_hour_of(2pm - 5pm)
```

Comment:
I like the idea of being able to specify some sort of conditional rule on the time you are free, but I'm not sure expressing it this way is the most ideal. Perhaps something more like:

```
   free 2pm - 5pm for 1 hour
```

would be more natural, but still parseable.